modules/installing-using-assisted-installer-oci.adoc

```diff
@@ -0,0 +1,158 @@
+* If you use a firewall and you plan to use a Telemetry service, you xref:../../installing/install_config/configuring-firewall.adoc#configuring-firewall[configured your firewall], so that {product-title} can access the sites required.
+* You configured an {oci} account to host the cluster. See link:https://docs.oracle.com/en/cloud/get-started/subscriptions-cloud/csgsg/create-users-and-assign-roles.html#GUID-1041AC78-9E43-42CF-B5A5-73F8B8FEB39A[Create Users and Assign Roles] in the Oracle documentation. 
+* If the cloud identity and access management (IAM) APIs are not accessible in your environment, or if you do not want to store an administrator-level credential secret in the `kube-system` namespace, you can manually create and maintain IAM credentials. See link:https://docs.oracle.com/en-us/iaas/Content/Identity/Concepts/overview.htm[Overview of Identity and Access Management] in the Oracle documentation. 
+* You set up your cloud infrastructure, such as compute, networking, and storage settings, before installing {product-title}. You must ensure that your infrastructure meets the minimum Assisted Installer link:https://access.redhat.com/documentation/en-us/assisted_installer_for_openshift_container_platform/2024/html-single/assisted_installer_for_openshift_container_platform/index#hardware[hardware requirements]. 
+
+Before you create your VMs, refer to {op-system-base-full} certified shapes, instance types, to identify the supported {oci} VM shapes. See link:https://catalog.redhat.com/cloud/detail/216977[Cloud instance types] on Red Hat Ecosystem Catalog portal.
+
+.Procedure
+
+. Log in to your link:https://cloud.oracle.com/a/[Oracle Cloud Infrastructure] account that has administrator privileges, and create a child compartment. Note the name and the link:https://docs.oracle.com/en-us/iaas/Content/libraries/glossary/ocid.htm[{ocid-first}] of the compartment. See link:https://docs.public.oneportal.content.oci.oraclecloud.com/en-us/iaas/process-automation/oci-process-automation/create-compartment.html[Create a compartment] in the Oracle documentation.
++
+[NOTE]
+====
+When creating the child compartment, specify the default parent compartment or any other parent compartment from the list.
+==== 
+
+. Create a bucket resource for your child compartment. Ensure that you specify your child compartment in the **Create in compartment** field for the bucket resource. Go to **Object Storage & Archive Storage > Buckets** and create a bucket, where **Bucket name** refers to your cluster's name.
+
+. From the link:https://console.redhat.com/openshift/assisted-installer/clusters/~new[**Install OpenShift with the Assisted Installer**] page on the Hybrid Cloud Console, generate the discovery ISO image by completing all the required Assisted Installer steps. Ensure that for the **Cluster Details** step, you complete the following fields:
```

Comment:
"by completing all the required Assisted Installer steps". This looks like "lazy writing". :-) It's like "Respond to all the prompts and wait for the process to complete" or "Fill in the fields and click OK". The AI is a wizard, so I would change this to a proper subprocedure:

. On the Hybrid Cloud Console[link], add a new cluster by using the Assisted Installer:

.. On the **Cluster details** page, specify the cluster name, base domain, OpenShift version,... and then click **Next**.
(comments about where to find base domain, which OCP version to use)

.. On the **Operators** page, click **Next**.

.. On the **Host discovery** page, click **Add hosts**. [note: See ISG for UI documentation.]

Reply:
I do no appreciate the "lazy writing" comment as a lot of work went into this. The history of changes on a Kbase article are not traceable but I can assure you that countless hours went into this work. I had to watch videos, navigate console, read Oracle's documentation, confer with many stakeholders, and so on.

Reply:
I think its excessive to mention each wizard step when only 2 of the steps are important in the context of creating a cluster on OCI.

Reply:
SME INPUT REQUIRED: Would breaking the flow of the procedure be useful to our customers. At the moment, all the steps fall into one module. Would breaking the prcoedure into 1+ modules be more distracting?